draw/spaceinvaders/spaceinvaders_test.go

```diff
@@ -2427,21 +2428,22 @@ func TestSVGGeneratedBySpaceInvaders(t *testing.T) {
 	t.Parallel()
 	mapOfColors := colors.MapOfColorThemes()
 	for k, expected := range mapExpectedSVG {
-		rec := httptest.NewRecorder()
+		w := &bytes.Buffer{}
 
 		h := md5.New()
 		io.WriteString(h, k)
 		md5Key := fmt.Sprintf("%x", h.Sum(nil)[:])
 
-		SpaceInvaders(rec, md5Key, mapOfColors["frogideas"][:2], 220)
-		if rec.Code != http.StatusOK {
-			t.Errorf("returned %v. Expected %v.", rec.Code, http.StatusOK)
-		}
-		if rec.Body.String() != expected {
+		SpaceInvaders(w, md5Key, mapOfColors["frogideas"][:2], 220)
+
+		// t.Log(w.String())
```

Comment:
Could you remove this unnecessary comments?

Reply:
Oops, will do in a sec.